.github/workflows/cluster.yaml

```diff
@@ -136,7 +136,7 @@ jobs:
             ./bin/zot-linux-amd64 serve test/cluster/config-minio3.json &
             sleep 10
             # run zb
-            bin/zb-linux-amd64 -c 10 -n 50 -o ci-cd http://localhost:8080
+            bin/zb-linux-amd64 -c 10 -n 50 -o ci-cd --src-cidr 127.0.0.0/8 http://localhost:8080
```

Comment:
We need a test for the "-i" option also, but not here.

Reply:
added